contrib_bots/bots/weather/weather.py

```diff
@@ -0,0 +1,74 @@
+# See readme.md for instructions on running this code.
+from __future__ import print_function
+import requests
+import json
+import os
+import sys
+from six.moves.configparser import SafeConfigParser
+
+DIRECTORY = os.path.dirname(os.path.realpath(__file__)) + '/'
+CONFIG_FILE = '.weather_config'
+RESPONSE_PATTERN = 'Weather in {}, {}:\n{} F / {} C\n{}'
+
+if not os.path.exists(DIRECTORY + CONFIG_FILE):
```

Comment:
One issue with putting this code at the "top level" (i.e. not inside of any class or function) is that it will run any time someone imports the `weather` module, this code will run.  In general, I think it's discouraged to put code at the "top level".

A better place for this code could be in an `__init__` function for the `WeatherHandler`.  What do you think?